server/objects/Feed.js

```diff
@@ -402,12 +402,12 @@ class Feed {
     this.xml = null
   }
 
-  buildXml() {
+  buildXml(originalHostPrefix) {
     if (this.xml) return this.xml
```

Comment:
It may be worthwhile to check if the cached `xml` is using the same url. 

Otherwise you could open an RSS feed on your local network to test, that caches the XML, then when accessing the RSS feed on your reverse proxy it still uses the cached version of the XML with your local ip address.

Reply:
I removed the xml cache - It is never used anyway, since a new Feed Object is created for each `getFeed` request. 

Caching for this needs to be handled by the API cache, which I need to revisit to make it more useful.

Reply:
Oh right, I forgot that I updated that to pull from the database on every request instead of keeping the feeds in memory. It would be better to handle that cache in a more standardized way anyway.